logdnaHanlderTest.py

```diff
@@ -0,0 +1,154 @@
+from BaseHTTPServer import BaseHTTPRequestHandler,HTTPServer
```

Comment:
filename is wrong - has to be `logdnaHandlerTest.py` - small mistake in writing...